jenkins/spark-tests.sh

```diff
@@ -59,12 +59,15 @@ tar zxf $SPARK_HOME.tgz -C $ARTF_ROOT && \
 PARQUET_PERF="$WORKSPACE/integration_tests/src/test/resources/parquet_perf"
 PARQUET_ACQ="$WORKSPACE/integration_tests/src/test/resources/parquet_acq"
 OUTPUT="$WORKSPACE/output"
+# need to disable pooling for udf test to prevent cudaErrorMemoryAllocation
```

Comment:
This is not how we tell users to setup the cluster to run with this feature.  We should be doing the same thing we ask users to do.  Partition the GPU memory between the python and JVM processes.  We also are doing this for all of the tests including TPCH.  We should only do it for the integration tests, and possibly split the test runs up so that we are only doing it for the UDF tests that need it.

Reply:
If we split the test cases, we need to add cunstomer marker of UDF test, like udf_cudf. Then we can use different setting to start the UDF test only with there 2 special settings. Would this be good to you?